internal/scanner/scanner.go

```diff
@@ -89,11 +89,16 @@ func (s *Scanner) ScanNamespace(nsName string) error {
 	// create PolicyReport
 	namespacedsReport := report.NewPolicyReport(namespace)
 	namespacedsReport.Summary.Skip = skippedNum
+	// old policy report to be used as cache
+	previousNamespacedReport, err := s.reportStore.GetPolicyReport(nsName)
+	if err != nil {
+		log.Error().Err(err).Msg("error getting PolicyReport from store")
```

Comment:
I'm getting always a log error here (I believe). We just add fields to the log, in this case the report name, or the namespace.